src/pages/partners/partners.mdx

```diff
@@ -12,4 +12,4 @@ hide_table_of_contents: true
 |          :-:           |   :-:    |-------------|---------|
 | DaoCloud          |   DaoCloud Enterprise 5.0    | DaoCloud Enterprise 5.0 (DCE 5.0) is an advanced and highly scalable cloud native operating system designed to provide a consistent and dependable experience across diverse infrastructures and environments. With its support for heterogeneous clouds, edge clouds, and multicloud management, DCE 5.0 offers unparalleled flexibility.       | https://www.daocloud.io/products/standard-edition/index.html     |
 | Huawei Cloud           |   IEF    | Edge computing platform built upon KubeEdge, provides extremely lightweight, edge intelligent, and powerful in terms of computing capabilities.        | https://www.huaweicloud.com/product/ief.html     |
-
+| KubeSphere |   KubeSphere Enterprise 4.0   | KubeSphere Enterprise (KSE) 4.0 is a new cloud-native operating system built by QingCloud Technologies, which has extraordinary epoch-making significance for KubeSphere. Based on the open and extensible architecture of KubeSphere LuBan, KubeSphere Enterprise 4.0 can easily achieve upstream and downstream linkage of applications, integrate various high-quality extensions from KubeSphere Marketplace anytime, and provide seamless business capabilities and highly consistent product experience.     | https://kubesphere.co/kse/   |
```

Comment:
Could you plz add some description related to edge computing capabilities in the description section? Thanks :) @faweizhao26

Reply:
> Could you plz add some description related to edge computing capabilities in the description section? Thanks :) @faweizhao26

Done.

Reply:
Okey, thanks!